### PR TITLE
Update create_aad_assets.sh for switch changes in aad-app-reg.sh

### LINKED ISSUE #2037: Error 'Invalid option: --swaggerui-redirecturl' when running 'make auth'
Remember to label the issue with the appropriate area, and also assign it to yourself if you plan to work on the issue in the near future.

**Describe the bug**
Error `Invalid option: --swaggerui-redirecturl.` when running `make auth`

**Steps to reproduce**

1. Set up a fresh set of env files
1. Run `make auth`

### PR DESCRIPTION
**Pull request by stuartleeks**
# Fixes #2037 

## What is being addressed

Describe the current behavior you are modifying.  Please also remember to update any impacted documentation.

## How is this addressed

Update `create_aad_assets.sh` with the renamed switch for `aad-app-reg.sh`

### CODE REVIEW

File: scripts/create_aad_assets.sh [hunk 1]
```diff
@@ -36,7 +36,7 @@ fi
 # Then register an App for the TRE Core.
 ./scripts/aad/aad-app-reg.sh \
   --name "${TRE_ID}" \
-  --swaggerui-redirecturl "https://${TRE_ID}.${LOCATION}.cloudapp.azure.com/api/docs/oauth2-redirect" \
+  --tre-url "https://${TRE_ID}.${LOCATION}.cloudapp.azure.com/api/docs/oauth2-redirect" \
```
Comment (marrobi) [inline scripts/create_aad_assets.sh:39]: I think this should just be `https://${TRE_ID}.${LOCATION}.cloudapp.azure.com`, not the full redirect path. As we add that on later in the script for both swagger and the new UI.

Comment (martinpeck) [reply scripts/create_aad_assets.sh:39]: Yep...

https://github.com/microsoft/AzureTRE/blob/main/scripts/aad/aad-app-reg.sh#L509

... so, `create_aad_assets.sh` was broken in two ways then?

Comment (marrobi) [reply scripts/create_aad_assets.sh:39]: Yes, although both on the same line. Expect something has got lost on a merge. I made the changes to the script and docs and ran through a couple of times.

Comment (marrobi) [reply scripts/create_aad_assets.sh]: @martinpeck I've remove that path. Good to approve?

**Review by martinpeck — APPROVED** [review]

**Comment by stuartleeks** [comment]
/test-destroy-env



`A resource with the ID "/subscriptions/***/resourceGroups/rg-***/providers/Microsoft.Network/privateDnsZones/privatelink.eventgrid.azure.net" already exists - to be managed via Terraform this resource needs to be imported into the State.`

https://github.com/microsoft/AzureTRE/runs/6925131582?check_suite_focus=true#step:3:2035

**Pull request merged**